### ISSUE TITLE
Bulk create

### ISSUE THREAD

**Issue by tomas-zeman** [rafaelsetragni/awesome_notifications#401]
Hi, 

I work on an app that creates irregular notifications for upcoming days (depending on calendar, user can have from 0 up to 10 events per day). It creates them using a loop one by one. I noticed it took at about 500 ms to execute createNotification method. The loop always creates 45 notifications, it means that the action could take apx 22 seconds. 

Is there a way to create those 45 notifications at once - in order to decrease the total time. Now user can close the app before all notifications are scheduled which could prevent the app from scheduling all notifications.

Or is it possible to call the method  in parallel.

**Comment by rafaelsetragni** [comment]
The notification creation itself take between 60-180ms, depending of what resources and layouts you've choosed to use to create your notification. The rest is all what the O.S. needs to display the notification. And you can take all these mesures inside this class:
https://github.com/rafaelsetragni/awesome_notifications/blob/master/android/src/main/java/me/carda/awesome_notifications/notifications/NotificationBuilder.java

To create what youre desiring, i need to understand what exactly what youre trying to do, and i cannot do it right now because im finishing the 0.7.0 version and the firebase companion plugin.

**Comment by rafaelsetragni** [comment]
Hi! Sorry for taking so long to respond to your issue on GitHub. I was focused on releasing the new 0.7.0 version of `awesome_notifications` and `awesome_notifications_fcm` and i didn't had time enough to do both. But now I can answer all your questions.

So now i'm asking you to recreate this topic using the new issue template. There's a lot of missing informations that i need to understand your problem, and all the instructions are already in the new issue template. Also, remember to check beforehand if your issue was posted by another user.

So I will automatically close all previous issues so far.
Sorry for the inconvenience and i will be waiting for your new issue request.

Thank you so much for your support!
